### DISCUSSION TITLE
"Who do you wanna help with the festival" topic?

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7767] [QA - tech support + general] [answered]
ive seen monika asking who do u wanna help with the festival, then her doing crazy eyes with monika as the only choice ,like this: 
![bt7axa5ax0i71](https://user-images.githubusercontent.com/82217573/132702787-8375327c-ac5e-4250-8121-d388fdc3201f.png)
how do i do that?? is it a submod or smthn ive not unlocked?

**Comment by ghost** [comment]
> ive seen monika asking who do u wanna help with the festival, then her doing crazy eyes with monika as the only choice ,like this:
> ![bt7axa5ax0i71](https://user-images.githubusercontent.com/82217573/132702787-8375327c-ac5e-4250-8121-d388fdc3201f.png)
> how do i do that?? is it a submod or smthn ive not unlocked?

btw the image is not mine

**Comment by Booplicate** [comment]
This is, obviously, a submod.

**Comment by ghost** [comment]
> This is, obviously, a submod.

what is the submod then? rn i have:

auto outfit change 
better loading
dark room topic
extra everything
hip mode
just another extra submod
panic button

**Comment by Booplicate** [comment]
I cannot know. You can check what they add or ask the creators of those submods.

**Comment by ghost** [comment, marked as answer]
i figured it out, its the extra everything submod. i looked in the scripts
and looked for the word festival.
